src/components/shop/shop-list-screen.js

```diff
@@ -20,6 +20,11 @@ export class ShopList extends Screen {
     }
 
     create() {
+        if (this.transientData.shop.mode == "shop") {
+            this.setStatsScreen(this.transientData.shopTitle + "buy");
+        } else {
+            this.setStatsScreen(this.transientData.shopTitle + "manage");
+        }
```

Comment:
`this.setStatsScreen(this.transientData.shopTitle + (this.transientData.shop.mode === "shop" ? "buy" : "manage"));`

can prob make this one line too